### ISSUE TITLE
Add articles repeatedly

### ISSUE THREAD

**Issue by radishzzz** [RSSNext/Follow#1018]
### Describe the bug

For the same article, after modifying and re-uploading, the article will be duplicated. What I modified is the suffix of the article, for example:
https://example.com/posts/001
https://example.com/posts/001/
https://example.com/posts/001.html
They are actually the same article, but Follow recorded them as three articles and could not be refreshed. I also tried deleting and re-adding the subscription, as well as uninstalling and reinstalling the software.
<img width="1201" alt="截屏2024-10-18 15 56 49" src="https://github.com/user-attachments/assets/106723cc-3042-4ecf-985c-78f0109c0f10">

### Feed Info

https://radishzz.cc/atom.xml

### App Type

Electron / App

### Version

0.0.1-alpha21

### Operating System

macOS

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by hyoban** [comment]
I think you'll have this problem with other RSS apps too, this is how RSS works

**Comment by radishzzz** [comment]
> I think you'll have this problem with other RSS apps too, this is how RSS works

Yes, as you said, other RSS apps also have this problem, but when I delete and re-add the subscription, the duplicate articles disappear. This means that they have a detection and de-duplication function, while Follow does not. I think this can be improved.

**Comment by hyoban** [comment]
We cannot clear the entries after you delete the subscription, because there are other users subscribing, and we cannot determine which historical entries should be deleted.

**Comment by radishzzz** [comment]
> We cannot clear the entries after you delete the subscription, because there are other users subscribing, and we cannot determine which historical entries should be deleted.

Well, this is a difficult point. But as the author of the feed, can I do this job? Just manage and delete the duplicate historical entries. I think this is a very useful function for this kind of troubling problem. Maybe other people will have this problem too.

**Comment by hyoban** [comment]
How about allowing the feed owner to reset the entire feed? Clear all historical entries and fetch them again

**Comment by radishzzz** [comment]
> How about allowing the feed owner to reset the entire feed? Clear all historical entries and fetch them again
> 
> https://discord.com/channels/1243823539426033696/1295208040043118665

Well, I think this is a simple and good way~

**Comment by radishzzz** [comment]
> How about allowing the feed owner to reset the entire feed? Clear all historical entries and fetch them again
> 
> https://discord.com/channels/1243823539426033696/1295208040043118665

Hello, I had updated the latest version and found the new function named Reset Feed. But actually It doesn't seem to work for my feed, the duplicate articles are still there after I reseted my feed https://app.follow.is/share/feeds/68090849347654656 .
To avoid mistakes, I even reseted it for three consecutive days. Now I think the feature may not work as expected.
